must-gather/Dockerfile

```diff
@@ -12,13 +12,13 @@ RUN curl --location --output velero.tgz https://github.com/openshift/velero/arch
     CGO_ENABLED=0 GOOS=linux go build -a -mod=mod -ldflags '-extldflags "-static"' -o /restic github.com/restic/restic/cmd/restic && \
     cd .. && rm -rf restic.tgz restic-${RESTIC_BRANCH}
 
-FROM registry.access.redhat.com/ubi8/go-toolset:1.17.10 AS gobuilder
+FROM registry.access.redhat.com/ubi8/go-toolset:latest AS gobuilder
```

Comment:
should be ubi 9?

Reply:
`registry.access.redhat.com/ubi9/go-toolset:latest`

Reply:
double checked at https://catalog.redhat.com/software/containers/ubi9/go-toolset/61e5c00b4ec9945c18787690?q=go-toolset&architecture=amd64&image=65a9105b2fc8e203ec90cc66&container-tabs=gti

Reply:
@mateusoliveira43 @kaovilai thank you.. fixing now